plugin/storage/grpc/shared/grpc_client.go

```diff
@@ -96,7 +109,7 @@ func (c *grpcClient) GetOperations(ctx context.Context, service string) ([]strin
 
 // FindTraces retrieves traces that match the traceQuery
 func (c *grpcClient) FindTraces(ctx context.Context, query *spanstore.TraceQueryParameters) ([]*model.Trace, error) {
-	stream, err := c.readerClient.FindTraces(context.Background(), &storage_v1.FindTracesRequest{
```

Comment:
I am not sure why this is using Background instead of passing the `ctx` arg directly. Is there any reasoning in git blame?  If not, I think it's a bug / oversight.

Reply:
That’s what I originally also thought. I think this is an oversight for `FindTraceIDs` because the server returns a response for that call: https://github.com/jaegertracing/jaeger/blob/master/plugin/storage/grpc/shared/grpc_server.go#L132. However, `FindTraces` does chunking: https://github.com/jaegertracing/jaeger/blob/master/plugin/storage/grpc/shared/grpc_server.go#L107. Hence the `context.Background()`.

I’m fairly confident `FindTraceIDs` context can be changed, not so sure about `FindTraces`. I could investigate in a separate PR. I’ll have to revisit the gRPC storage anyway because I’ll also hit this: https://github.com/jaegertracing/jaeger/issues/1763

What do you think, @yurishkuro?

Reply:
Not sure I follow how chunking is related to using Background().

Reply:
I’ve seen hanging plugins when doing bidirectional calls. But here is no bidirectional communication. I’ve got plenty confused here. Code adapted to use the original context.